quarterback-packages/types/types/doc.d.ts

```diff
@@ -29,6 +29,7 @@ export type IGetDocumentResponse = ManuscriptDocWithSnapshots
 export interface ICreateDocRequest {
   manuscript_model_id: string
   project_model_id: string
+  user_model_id: string
```

Comment:
What is this used for?

Reply:
This is used for when a PE views takes action on changes (approve/reject) he would know who made this change

Reply:
Got it.